pkgs/by-name/el/eloquent/package.nix

```
@@ -0,0 +1,87 @@
+{
+  stdenv,
+  lib,
+  fetchgit,
+  meson,
+  glib,
+  gjs,
+  ninja,
+  gtk4,
+  gsettings-desktop-schemas,
+  wrapGAppsHook4,
+  desktop-file-utils,
+  gobject-introspection,
+  glib-networking,
+  pkg-config,
+  libadwaita,
+  appstream,
+  blueprint-compiler,
+  gettext,
+  libportal-gtk4,
+  languagetool,
+  fasttext,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "eloquent";
+  version = "1.2";
+
+  # fetchFromGitHub did not fetch submodules
+  src = fetchgit {
+    url = "https://github.com/sonnyp/Eloquent.git";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-I4AQZl1zoZPhOwDR1uYNJTMRq5vQHPvyimC8OUAe+vY=";
+    fetchSubmodules = true;
```

```suggestion
    fetchSubmodules = true;
    hash = "sha256-I4AQZl1zoZPhOwDR1uYNJTMRq5vQHPvyimC8OUAe+vY=";
```
Nit: it's better to have `fetchSubmodules = true` above the hash as it has an effect on it.